segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/SegmentContainerCollection.java

```diff
@@ -82,4 +86,26 @@ public SegmentContainerCollection(SegmentContainerRegistry segmentContainerRegis
 
         return resultFuture;
     }
+
+    /**
```

Comment:
I don't know if this is the best way of doing it. Why don't we execute from the admin CLI one `flushToStorage` command per container? We may have more granularity, in case we want to execute `flushToStorage` only on a specific container.

Reply:
I think it's good without it. This command is useful in data recovery which requires the entire data from all segment containers to be flushed.